addons/website/static/src/snippets/s_process_steps/options.js

```diff
@@ -108,8 +108,10 @@ options.registry.StepsConnector = options.Class.extend({
             const stepHeightDifference = stepPaddingTop - nextStepPaddingTop;
             this.hCurrentStepIconHeight = stepMainElementRect.height / 2;
             this.hNextStepIconHeight = nextStepMainElementRect.height / 2;
+            const widthOffset = stepMainElementRect.width / 2;
 
-            connectorEl.style.left = `calc(50% + ${stepMainElementRect.width / 2}px + 16px)`;
+            connectorEl.style.left = `calc(50% + ${widthOffset}px + 16px)`;
+            connectorEl.style.right = `calc(50% + ${widthOffset}px + 16px)`;
```

Comment:
I don't understand how it could make sense to have `right` **and* `left` properties both higher than 50%. If it does make sense, this needs code comments.